torch/fx/experimental/symbolic_shapes.py

```diff
@@ -96,6 +97,41 @@ def uninteresting_files():
 class ConstraintViolationError(RuntimeError):
     pass
 
+class _SingletonInt(sympy.AtomicExpr):
+    def __init__(self, val, factor):
+        self.val = val
+        self.factor = factor
```

Comment:
what's a factor?

Reply:
oh coefficient is probably a better name, I've updated "factor" -> "coeff" everywhere